packages/e2e-tests/src/specs/dashboard/dashboard.js

```diff
@@ -42,6 +42,27 @@ describe('Stories Dashboard', () => {
     await takeSnapshot(page, 'Stories Dashboard', { percyCSS });
   });
 
+  it('should be able to skip to main content of Dashboard', async () => {
+    await visitDashboard();
+
+    // If there are no existing stories, the app goes to the templates page instead.
+    // Account for both here, but then force-visit the dashboard.
```

Comment:
To account for the "force-visit" part this is missing the

```js
await expect(page).toClick('[aria-label="Main dashboard navigation"] a', {
  text: 'Dashboard',
});
```

part

Reply:
Thanks! Realized I actually don't want to force anything and we just want to see what happens with the navigation by keyboard once we're in the Stories Dashboard. Fixed this up and added better comments.